discord/public_flags.py

```diff
@@ -0,0 +1,96 @@
+from .enums import UserFlags
+
+
+class PublicFlags:
+    """Wraps up the Discord public flags.
+
+    .. versionadded :: 1.3.4
```

Comment:
```suggestion
    .. versionadded :: 1.4
```

Also these tags are generally added at the bottom of the operation table but above the attribute table. See `flags.py` for an example.